packages/zero-schema/src/builder/schema-builder.ts

```diff
@@ -24,29 +23,16 @@ export type Schema = {
  * You can assign them to any value you like. E.g.,
  *
  * ```ts
- * createSchema(1, {rsdfgafg: table('users')...}, {sdfd: relationships(users, ...)})
+ * createSchema({rsdfgafg: table('users')...}, {sdfd: relationships(users, ...)})
  * ```
- *
- * @param version The version of the schema. Only needs to be incremented
- * when the backend Postgres schema moves forward in a way that is not
- * compatible with the frontend. As in, if:
- * 1. Columns are removed
- * 2. Optional columns are made required
- *
- * Adding columns, adding tables, adding relationships, making
```

Comment:
This documentation was not quite right.

It should be:

Adding optional columns